src/mpi/romio/include/mpio.h.in

```diff
@@ -12,6 +12,12 @@
 
 #include "mpi.h"
 
+#if defined(MPICH_C_HAVE_VISIBILITY)
+#define MPICH_API_PUBLIC __attribute__((visibility ("default")))
```

Comment:
I agree we could probably clean up the macros/defines to be less MPICH-specific. I had meant to do that once I copied to the various sub-components. I'm going to whip up a new version that addresses this. Shouldn't have any functional changes.